### DISCUSSION TITLE
WMS problem

### DISCUSSION THREAD

**Discussion by mcecilia-cl** [opengisch/QField#4336] [Q&A]
I'm having problems with wms in Qfield. I've tried with several wms and only a few worked. For exemple, OpenStreetMap works fine but not OpenTopoMap. I'm particularly interested in using french IGN wms, but I'm having no success.

Thank you,

**Comment by nirvn** [comment]
@mcecilia-cl , sharing sample projects here would help moving this discussion forward.

**Comment by geguill** [comment]
Bonjour,
je rebondis sur le problème évoqué par NIRVN.
Mon projet comporte des couches WMS en provenance de l'IGN France, QField Androïd (9) refuse de les ouvrir alors que QField Windows n'a pas de soucis.
Je joins le projet et une copie d'écran du refus d'ouvrir les couches.
Merci de votre aide.
geguill
[ASP_1_plus_light.zip](https://github.com/opengisch/QField/files/11890568/ASP_1_plus_light.zip)
![Screenshot_20230628-074343](https://github.com/opengisch/QField/assets/98580457/2d8a86a6-7e26-4c6d-9330-844e565397b2)

**Reply by nirvn** [reply]
I've been able to load both layers here. Proof of life:

![image](https://github.com/opengisch/QField/assets/1728657/ed3fdb48-3daa-4c2d-be43-68c6871aa01a)

![image](https://github.com/opengisch/QField/assets/1728657/daf958e8-84b9-4369-ab79-a461ae348cb3)

Have you tried to remove and re-install QField? Users that have had QField installed for a relatively long period of time and updated from one version to the other have reported such issue; re-installing fixed it for them.

**Reply by geguill** [reply]
Hélas !
J'ai désinstallé QField puis installé. Pas de changements !

**Reply by nirvn** [reply]
@geguill , do you have a VPN or proxy on your device that would prevent connection to the WMS server?

**Reply by geguill** [reply]
Je ne sais pas, j'interroge le service technique et je vous tiens informé.
Merci

**Reply by geguill** [reply]
Bonjour,
j'ai interrogé le service IGN France, voici leur réponse :
"Nous vous invitons à vous rapprocher du support de l'application, l'incident ne semble pas provenir de nos flux directement puisque vous arrivez à les afficher sur SIG et une autre version de l'application. 
Il est possible que cela soit lié à un problème de certificat sur la version Android, il faut que l'éditeur l'ajoute et propose une mise à jour à ses utilisateurs : https://geoservices.ign.fr/2023-04-certificat-ssl"

Est-ce une piste ?
Merci

**Reply by nirvn** [reply]
@geguill , peux tu essayer d installer un APK beta ici: https://github.com/opengisch/QField/pull/4407#issuecomment-1612654405

La prochaine version de QField sur Android fait une grosse mise a jour openssl, ça peut aider.

**Reply by geguill** [reply]
au risque de passer pour un débutant nul, j'ai essayé l'APK préconisée, impossible de l'installer sur ma tablette Galaxy Tab Active2 équipée d'Android 9.
J'ai vu que d'autres architectures existent, laquelle je peux prendre ?
Merci
![Capture d'écran 2023-06-29 145643](https://github.com/opengisch/QField/assets/98580457/68107cd9-e3fb-4d30-a52f-33ee3770c8ab)

**Reply by nirvn** [reply]
@geguill , pour ta tablet ca serait arm64-android

**Comment by VxTedxV** [comment]
related to #4243 ?
I tried IGN WMS at different times : sometimes works, sometimes don't... that's why I think it's related to IGN provider...

**Reply by mcecilia-cl** [reply]
For me, it is not working at all, never. But I tried the same Qfield project within another device (a pear's smartphone) and it worked fine. So I wonder if it is a matter of device compatibility...

**Reply by geguill** [reply]
Oui, c'est bien ça. J'ai testé avec une tablette Samsung Galaxy Active 3 et pas de problème.
Bonne journée
Merci à tous

**Comment by mcecilia-cl** [comment]
J'ai ajouté le certificat dans mon portable (Android 7.0), désinstallé et réinstallé Qfield, et les flux IGN marchent toujours pas. Je crois qu'il ne reste plus qu'attendre la grosse mise à jour openssl de Qfield cité par nirvn. Merci en tout cas,

> Bonjour, j'ai interrogé le service IGN France, voici leur réponse : "Nous vous invitons à vous rapprocher du support de l'application, l'incident ne semble pas provenir de nos flux directement puisque vous arrivez à les afficher sur SIG et une autre version de l'application. Il est possible que cela soit lié à un problème de certificat sur la version Android, il faut que l'éditeur l'ajoute et propose une mise à jour à ses utilisateurs : https://geoservices.ign.fr/2023-04-certificat-ssl"
> 
> Est-ce une piste ? Merci

**Reply by nirvn** [reply]
@mcecilia-cl , as tu essayé d installer un APK dans le lien que j ai fourni? Si tu as un CPU ARM 32 bit, installes arm-neon-android. Si c 64 bit, arm64-android. Donnes nous des nouvelles.

**Reply by geguill** [reply]
J'ai pu installer la version arm neon, mais le problème persiste...
J'attends tranquillement une mise à jour officielle.
Encore merci

**Reply by mcecilia-cl** [reply]
Non, je n'ai pas essayé parce que ça dépasse mes connaissances (je ne sais pas ce que c'est qu'un APK, oups). Comme @geguill , j'attendrai patiemment. Mais c'est bien gentil de demander @nirvn . Merci beaucoup,

**Comment by geguill** [comment]
Bonjour,
finalement c'est la tablette Samsung tab active 2 qui pose problème ; la version active 3 accepte les flux de l'IGN.
Merci à tous
geguill

**Comment by mcecilia-cl** [comment]
Pour info, de mon côté c'est un téléphone portable Samsung Galaxy S6 qui présente le problème.
    Le lundi 10 juillet 2023 à 14:16:31 UTC+2, geguill ***@***.***> a écrit :  
 
 


Bonjour,
finalement c'est la tablette Samsung tab active 2 qui pose problème ; la version active 3 accepte les flux de l'IGN.
Merci à tous
geguill
